Kentico.Kontent.Delivery.Abstractions/DeliveryClientExtensions.cs

```diff
@@ -65,5 +65,16 @@ public static Task<IDeliveryTaxonomyListingResponse> GetTaxonomiesAsync(this IDe
         {
             return client.GetTaxonomiesAsync(parameters);
         }
+
+        /// <summary>
+        /// Returns languages.
```

Comment:
Could we be a bit more specific? Something like "returns all active languages assigned to a given project and matching the optional filtering parameters."...